sdk/eventhub/eventhubs-checkpointstore-tables/src/tableCheckpointStore.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CheckpointStore, PartitionOwnership, Checkpoint } from "@azure/event-hubs";
+/*
+import {  TableClient } from "@azure/data-tables";
+*/
```

Comment:
Let's remove the commented out code for now. You can add it back in when you start the implementation in another PR!